source/mainnet/smart-contracts/tutorials/counter/setup-env.rst

```diff
@@ -0,0 +1,101 @@
+.. include:: ../../../variables.rst
+.. _setup-env:
+
+=================================
+Setup the development environment
+=================================
```

Comment:
Why are we repeating this? Don't we already have the exact same instructions elsewhere where we could link to them?

Reply:
So I have completed this, also for the Piggy Bank. All tutorials now point to the one setup dev environment topic.